src/sagemaker/amazon/kmeans.py

```diff
@@ -43,11 +43,6 @@ class KMeans(AmazonAlgorithmEstimatorBase):
     )
     epochs = hp("epochs", gt(0), "An integer greater-than 0", int)
     center_factor = hp("extra_center_factor", gt(0), "An integer greater-than 0", int)
-    eval_metrics = hp(
```

Comment:
I remember doing some more digging back during #1480 - https://docs.aws.amazon.com/sagemaker/latest/dg/k-means-api-config.html seems to contradict the internal backlog item we had, so let's keep `eval_metrics` in KMeans and RCF